edx_rest_framework_extensions/__init__.py

```diff
@@ -1,3 +1,3 @@
 """ edx Django REST Framework extensions. """
 
-__version__ = '4.0.2'  # pragma: no cover
+__version__ = '4.0.3'  # pragma: no cover
```

Comment:
**Reviewer Note:**
We are in a strange state where this should really be 3.0.1, and we already have the issue where 4.0.1 was a breaking change and should have been 5.0.0.  Given that, since the Lahore team is actively doing all these upgrades for Django 2.2, I am planning on releasing this as a fix on v4, namely 4.0.3.

Let me know if anyone disagrees with this decision.